src/editors/string.js

```diff
@@ -31,7 +31,7 @@ JSONEditor.defaults.editors.string = JSONEditor.AbstractEditor.extend({
 
     this.input.value = sanitized;
     
-    // If using SCEditor, update the WYSIWYG
+/*    // If using SCEditor, update the WYSIWYG
```

Comment:
Please remove the comments ;)

Reply:
>
> I have kept the original code in src/editors/string.js for now (commented out) so don't merge it yet. 😉

I will removed it, if everyone agrees to the changes. As it changes how you configure the "plugins". (Now uses `JSONEditor.defaults.options` and schema options instead of `JSONEditor.plugins`)

(I'm still tweaking the code, and I keep the comments as reference until I'm done.)

Reply:
> I will removed it, if everyone agrees to the changes.
> ...
> (I'm still tweaking the code, and I keep the comments as reference until I'm done.)

You can't really understand the changes in the diff, if you do not see which lines are removed.

Reply:
Good point. I have committed a version without the comments.